featuretools/computational_backends/feature_set.py

```diff
@@ -70,9 +70,9 @@ def feature_trie(self):
         then used for all subsequent calls.
 
         The edges of the trie are RelationshipPaths and the values are tuples of
-        (bool, set[str], set[str]). The bool represents whether the full entity df is needed at
+        (bool, set[str], set[str]). The bool represents whether the full df is needed at
```

Comment:
might be worth lengthening `df` to `dataframe` now that it doesn't have the extra `entity` descriptor